src/lib.rs

```diff
@@ -47,12 +47,11 @@ pub use circuit_sequencer_api::INITIAL_MONOTONIC_CYCLE_COUNTER;
 
 // #[cfg(test)]
 pub mod helper;
+pub(crate) mod tests;
```

Comment:
nit: any reason for this diff?

Reply:
Actually not - tried to group "test" modules together